app/cdap/components/FooterPanel/Components/common/TabWrapper/index.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box } from '@material-ui/core';
+import { ITabWrapperProps } from 'components/FooterPanel/Components/common/TabWrapper/types';
+import React from 'react';
+import styled from 'styled-components';
+
+const SmallBox = styled(Box)`
+  &&& {
```

Comment:
it seems like the &&& is being overused - this should only be used to override specificity

Reply:
Done.